### PR TITLE
feat(styling): Align with website closes (#4)

### PR DESCRIPTION
**Pull request by fabioelia**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "build": "gatsby build",
     "clean": "gatsby clean",
-    "start": "gatsby develop -o"
+    "start": "gatsby develop -o --host=0.0.0.0"
```
Comment (trybick) [inline package.json]: curious what the advantage is?

File: src/components/layout.js [hunk 2]
```diff
@@ -4,61 +4,217 @@ import styled, { createGlobalStyle } from 'styled-components';
...
-    font-family: 'system';
+    font-family: var(--font-default);
     margin: 0;
     text-rendering: optimizeLegibility;
     -webkit-font-smoothing: antialiased;
-    color: rgba(0, 0, 0, 0.8);
     min-height: 100vh;
     position: relative;
     font-size: 1.6rem;
+    // background: linear-gradient(-45deg, #2d2d2d, #222);
```
Comment (trybick) [inline src/components/layout.js]: nit - commented code

**Review by trybick — APPROVED** [review]
omg this is amazing 😍

File: package.json (hunk 1 above)
Comment (fabioelia) [reply package.json]: so veiko could see it in our local network ;)

**Pull request merged**